src/services/file/caching.ts

```diff
@@ -0,0 +1,49 @@
+import { hoursToSeconds } from 'date-fns';
```

Comment:
Looks like this could be just a Redis service? Maybe like defining a constructor with `expiringTime` `key`, `buildKey`, `callback` and it could be reuse for anything, right?